src/clef.ts

```diff
@@ -2,31 +2,11 @@
 // MIT License
 // Co-author: Benjamin W. Bohl
 
-import { Glyph } from './glyph';
 import { Stave } from './stave';
 import { StaveModifier, StaveModifierPosition } from './stavemodifier';
 import { Tables } from './tables';
 import { Category } from './typeguard';
-import { defined, log, upperFirst } from './util';
-
-export interface ClefType {
-  code: string;
-  line: number;
-}
-
-export interface ClefAnnotatiomType extends ClefType {
```

Comment:
Ha... good thing we no longer need this interface! It had a typo that we never caught....

Reply:
You are right!